java-shared-dependencies/dependency-convergence-check/pom.xml

```diff
@@ -30,6 +40,10 @@
       <version>1.5.13</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.maven.shared</groupId>
+      <artifactId>maven-shared-utils</artifactId>
```

Comment:
(A minor thing) Did you find this line is needed? I think dependencyManagement section is enough.

Reply:
Ah, let me double check this. I think you're right.

Reply:
Yep, removed. Thanks!